leakage-test/leakage-test_shacl.ttl

```diff
@@ -0,0 +1,145 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix leakage-test: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/leakage-test> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+leakage-test:LeakageTestShape
+    a sh:NodeShape ;
+    sh:property [ sh:minCount 1 ;
+        sh:maxCount 1 ;
+        sh:node leakage-test:MachineParameterShape ;
+        sh:path leakage-test:MachineParameter ] ,
+    [ sh:minCount 1 ;
+        sh:maxCount 1 ;
+        sh:node leakage-test:ProcessParameterShape ;
+        sh:path leakage-test:ProcessParameter ] ,
+    [ sh:minCount 1 ;
+        sh:maxCount 1 ;
+        sh:node leakage-test:CodingDownholderShape ;
+        sh:path leakage-test:CodingDownholder ] ,
+    [ sh:minCount 1 ;
+        sh:maxCount 1 ;
+        sh:node leakage-test:CodingTestAdapterShape ;
+        sh:path leakage-test:CodingTestAdapter ] ,
+    [ sh:minCount 1 ;
+        sh:maxCount 1 ;
+        sh:node leakage-test:CodingRFIDShape ;
+        sh:path leakage-test:CodingRFID ] ,
+    [ sh:minCount 1 ;
+        sh:maxCount 1 ;
+        sh:node leakage-test:MaximumPressureShape;
+        sh:path leakage-test:MaximumPressure ] ,
+    [ sh:minCount 1 ;
+        sh:maxCount 1 ;
+        sh:node leakage-test:MinimumPressureShape;
+        sh:path leakage-test:MinimumPressure ]; 
+    sh:targetClass leakage-test:LeakageTest .
+
+leakage-test:MachineParameterShape
+    a sh:NodeShape ;
+    sh:property [ sh:minCount 1 ;
+	sh:maxCount 1 ] ;
+    sh:targetClass leakage-test:MachineParameter .
```

Comment:
Each targetClass has to exist as OWL class in the ontology file.
Please add it there for all targetClasses.
See also: https://github.com/GAIA-X4PLC-AAD/ontology-management-base/pull/73

Reply:
I added the classes.